### PR TITLE
Eunit cleanup

### LINKED ISSUE #96: eunit tests rewrite as pure coverage

### PR DESCRIPTION
**Pull request by c-bik**
fixes #96

### CODE REVIEW

File: test/oranif_eunit.erl [hunk 1]
```diff
@@ -0,0 +1,1122 @@
...
+        dpiCall(Safe, stmt_getQueryValue, [Stmt, 1]),
+    ?assertEqual(12345.0, dpiCall(Safe, data_get, [Result])),
+    ?assertEqual(Type, 'DPI_NATIVE_TYPE_DOUBLE'),
+    ?assertEqual(Query_cols, 5),
+    dpiCall(Safe, data_release, [Result]),
+    dpiCall(Safe, stmt_release, [Stmt]).
+
+create_insert_select_drop({Safe, _Context, Conn}) -> 
+    ?EXEC_STMT(Conn, <<"select 12345, 2, 4, 8.5, 'miau' from dual">>),
+    ?EXEC_STMT(Conn, <<"select 12345, 2, 4, 844.5, 'miau' from dual">>),
```
Comment (acautin) [inline test/oranif_eunit.erl]: What is being tested in this lines ? No side effect so they probably could be removed without changing the result as any error is catch on the Macro `EXEC_STMT` or is my understanding not correct ?

Comment (c-bik) [reply test/oranif_eunit.erl]: @KarlKeiser can please take care of it in your branch?

Comment (KarlKeiser) [reply test/oranif_eunit.erl]: You're right, they're not needed anymore, I removed them (in branch `karl-eunit_cleanup_1`)

**Comment by c-bik** [comment]
Closing this one.
@KarlKeiser Please open a new "eunit cleanup" PR from `eunit_cleanup` branch to `master`

**Pull request closed without merging**